tests/test_remove_mon_from_cluster.py

```diff
@@ -0,0 +1,91 @@
+"""
+A Testcase to remove mon from
+when I/O's are happening.
+
+Polarion-ID- OCS-355
+
+"""
+
+import logging
+import re
+import random
+import time
+from ocs import ocp, constants
+from ocsci.testlib import tier4, ManageTest
+from ocsci.config import ENV_DATA
+from resources import pod
+from tests.helpers import ceph_health_check, run_io_with_rados_bench
+
+
+log = logging.getLogger(__name__)
+
+DEP = ocp.OCP(
+    kind=constants.DEPLOYMENT, namespace=ENV_DATA['cluster_namespace']
+)
+POD = ocp.OCP(
+    kind=constants.POD, namespace=ENV_DATA['cluster_namespace']
+)
+
+
+def get_mons_from_cluster():
+    """
+    Getting the list of mons from the cluster
+
+    Returns:
+        available_mon (list): Returns the mons from the cluster
+    """
+
+    ret = DEP.get(
+        resource_name='', out_yaml_format=False, selector='app=rook-ceph-mon'
+    )
+    available_mon = re.findall(r'[\w-]+mon-+[\w-]', ret)
+    return available_mon
+
+
+def remove_mon_from_cluster(mon):
+    """
+    Removing the mon pod from deployment
+    """
+    assert DEP.delete(resource_name=mon), f"Failed to delete mon {mon}"
```

Comment:
Please add `logger.info` message so it is clear what is happening.

Reply:
I see you forgot to add logger as Filip asked.